src/app/shared/state/token.service.ts

```diff
@@ -68,15 +68,19 @@ export class TokenService {
     this.tokenStore.updateRoot({ gitHubOrganizations: gitHubOrganizations });
   }
 
-  getGitHubOrganizations(token: String) {
-    const httpClient = new HttpClient(this.httpBackend);
-    const httpOptions = {
-      headers: new HttpHeaders({
-        'Content-Type': 'application/json',
-        Authorization: 'token ' + token
-      })
-    };
-    const getOrganizationUrl = 'https://api.github.com/user/orgs';
-    httpClient.get(getOrganizationUrl, httpOptions).subscribe(gitHubOrganizations => this.setGitHubOrganizations(gitHubOrganizations));
+  getGitHubOrganizations(token: string | null) {
+    if (token) {
+      const httpClient = new HttpClient(this.httpBackend);
+      const httpOptions = {
+        headers: new HttpHeaders({
+          'Content-Type': 'application/json',
+          Authorization: 'token ' + token
+        })
+      };
+      const getOrganizationUrl = 'https://api.github.com/user/orgs';
```

Comment:
Not a fan about this occurring in the UI as opposed to in the web-service, it seems a bit too one-of-a-kind.

Reply:
This PR just checks the null-checks the token.  Can put this in the webservice some other time.